### PR TITLE
Updated german translation

### PR DESCRIPTION
**Pull request by malekairmaroc7**
I updated some translation.

#### What's changed?

_Tell us what you've changed._

#### Where have you tested it?

_Tell us where have you tested it._

*** 
#### Pull Request status
- [ ]  This PR has been tested using the provided devkitPro, devkitARM, and EasyGL2D.

### CODE REVIEW

File: romsel_aktheme/nitrofiles/language/lang_de/language.txt [hunk 1]
```diff
@@ -9,64 +9,64 @@ Paste = Einfügen
...
 
 [cheats]
 button = Cheats
 title = Cheats-Menü
 confirm = Bestätigen
-Cancel = Abbr.
+Cancel = Abbrechen
 info = Eigenschaften
-deselect = Deselect all
+deselect = Auswahl rückgangig machen
```
Comment (R-YaTian) [inline romsel_aktheme/nitrofiles/language/lang_de/language.txt:26]: Well, this strings too long for deselect button. And cannot be displayed well.

Comment (malekairmaroc7) [reply romsel_aktheme/nitrofiles/language/lang_de/language.txt:26]: I will update it. Thank you for letting me know!

Comment (R-YaTian) [reply romsel_aktheme/nitrofiles/language/lang_de/language.txt:26]: > Okay. Thanks for updating! I will try keeping translations to a minimum

Why not use 'Alle abwählen' for alt?

Comment (malekairmaroc7) [reply romsel_aktheme/nitrofiles/language/lang_de/language.txt:26]: Great idea lmao

**Comment by malekairmaroc7** [comment]
You may merge it now.

**Comment by R-YaTian** [comment]
> You may merge it now.

@RocketRobz

**Pull request merged**